features/admin.users.v1/components/user-impersonation-action.tsx

```diff
@@ -108,6 +108,8 @@ export const UserImpersonationAction: FunctionComponent<UserImpersonationActionI
         state.config.ui.features?.users);
     const accountAppClientID: string = useSelector((state: AppState) =>
         state.config.deployment.accountApp.clientID);
+    const authenticatedUserTenanted: string = useSelector((state: AppState) => state?.auth?.username);
```

Comment:
WDYT?

```suggestion
    const impersonatorUsername: string = useSelector((state: AppState) => state?.auth?.username);
```

Reply:
Since the authenticated user is not necessarily the impersonator, avoided addressing the authenticated user as the impersonator directly.